deploy/olm-catalog/qdrouterd-operator/0.1.0/qdrouterd-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -12,18 +11,28 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - version: v1alpha1
-      kind: Qdrouterd
-      description: An instance of Qdrouterd
+    - description: An instance of Qdrouterd
       displayName: Qdrouterd
+      kind: Qdrouterd
```

Comment:
Same comment here, and for alm examples

Reply:
changes made to fix those unnecessary write spaces and position movement.